frontend/src/pages/EventPage/EventPage.js

```diff
@@ -0,0 +1,80 @@
+import React, {useEffect, useState} from 'react'
+import {useParams} from "react-router-dom";
+import { useAuth } from "../../auth/useAuth"
+
+import { Typography, Grid } from '@mui/material';
+import CommentSection from "../../common/CommentSection"
+
+function EventPage() {
+  
+  let { id } = useParams();
+
+  const [state, setState] = useState({
+    error: null,
+    isLoaded: false,
+    event: [] 
+  })
+
+  const { token } = useAuth()
+  
+  useEffect(() => {
+
+    const fetchArgs = {
+      method: "GET",  
+    }
+
+    if (token) fetchArgs.headers = {Authorization: "Bearer " + token}
+    fetch('/api/event/' + id, fetchArgs)
+      .then((response) => response.json())
+      .then((data) => {
+        setState({error: null, isLoaded: true, event: data})
+      },
+        error => {
+          setState({error: error})
+        }
+      )
+  }, [id, token])
+
+  const {error, isLoaded, event} = state
+
+  if (error) {
+    return <div>Error: {error.message}</div>
+  } else if (!isLoaded) {
+    return <div>Loading...</div>
+  } else {
+    console.log(event)
```

Comment:
Can we remove this part if there is no need for it?

Reply:
I have removed the console log lines from art item and event pages.